docs/kpod-logout.1.md

```diff
@@ -0,0 +1,49 @@
+% kpod(1) kpod-logout - Simple tool to logout of a registry server
+% Urvashi Mohnani
+# kpod-logout "1" "August 2017" "kpod"
+
+## NAME
+kpod-logout - Logout of a container registry
+
+## SYNOPSIS
+**kpod logout**
+**REGISTRY**
+[**--help**|**-h**]
```

Comment:
Should we add a --all option, to logout all logins?

Reply:
we can, it is a good option to have in my opinion.

Reply:
@rhatdan do we want to add an --all option?

Reply:
Yes